languages/java/oso/src/test/java/com/osohq/oso/PolarTest.java

```diff
@@ -47,6 +47,18 @@ public static String myStaticMethod() {
     public String myReturnNull() {
       return null;
     }
+
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
Would this work if it was `equals(MyClass other)` instead?

Reply:
That would be overloading instead of overriding -- might work here since we only pass other instances of `MyClass` into this `equals()` method, but I'm not sure there's a reason to prefer that over what's currently here.

Reply:
Oh right. I was thinking it would be more normal for someone to implement equals for specific instance instead of the generic equals for any object.